draft-ietf-quic-transport.md

```diff
@@ -851,6 +851,11 @@ limits are set in the transport parameters (see
 using MAX_STREAMS frames ({{frame-max-streams}}). Separate limits apply to
 unidirectional and bidirectional streams.
 
+If a max_streams transport parameter or MAX_STREAMS frame carries a value
+greater than 2^60, which implies a maximum stream ID that cannot be expressed,
```

Comment:
```suggestion
greater than 2^60, which implies a maximum stream ID that cannot be expressed in later frames (see {{frame-formats}}),
```

Reply:
I don't think that we need to say this.  It's strictly more correct, but the extra words only obscure the meaning.  Isn't it obvious that we're only talking about QUIC?

Reply:
Agreed.

Reply:
My addition was (perhaps poorly) trying to articulate the difference between expressing it in transport params/MAX_STREAMS vs. in other frames. Saying that it's not expressible isn't accurate -- it was "expressed" when we received it -- what we want to say is that it's not expressible in other frames, specifically as a varint. 

Maybe what I should have suggested is 
"cannot be expressed as a variable-length integer (see {{integer-encoding}})."